pkg/disk/export.go

```diff
@@ -0,0 +1,138 @@
+package disk
```

Comment:
please create separate package called vm-export

Reply:
Why it's needed? We already have `export.go` that is the purpose of it to interact with VMExport API.